server/bin/pbench-index.py

```diff
@@ -331,30 +376,30 @@ def main(options, name):
                             idxctx.es, actions, fp, idxctx.logger, idxctx._dbg
                         )
                 except UnsupportedTarballFormat as e:
-                    idxctx.logger.warning("Unsupported tar ball format: {}", e)
-                    tb_res = 4
+                    idxctx.logger.warning(
+                        "{}: {}", error_codes["TB_META_ABSENT"].message, e
+                    )
+                    tb_res = error_codes["TB_META_ABSENT"].value
```

Comment:
Interesting. While this is a faithful representation of the previous error code here, I'm not sure that it was really accurate. The description references the metadata file, but the code in the try block seems most likely to fail attempting to open the tarball or maybe constructing the generator. Does it even reference the metadata file?

Pete, should this maybe really have been "problem reading tar file" rather than referring to metadata in the first place? It seems that, regardless of the error code, the old log message may have been more meaningful because it really doesn't match with the comment description of error 4.

I guess I kinda opened this can of worms... but maybe we can at least straighten out the worms  a bit before we close it again.

Reply:
@portante can you please clear this confusion